grails-app/domain/au/org/emii/aatams/CodeMap.groovy

```diff
@@ -7,7 +7,7 @@ package au.org.emii.aatams
  */
 class CodeMap  {
```

Comment:
This looks like straight Java, should it have a java file extension? I am surprised the grails compiler is able to handle it.